docker-compose.yml

```diff
@@ -93,6 +94,31 @@ services:
     networks:
       - reefscan
 
+  backtracking:
+    image: reef-explorer-backtracking-dev
+    command: yarn dev-backtracking
+    build:
```

Comment:
we are reusing image from crawler (`reef-explorer-crawler-dev`) so no need to build here, should be fine to remove `build`.